src/screens/Profile/Header/Shell.tsx

```diff
@@ -59,6 +59,7 @@ let ProfileHeaderShell = ({
           {
             uri: profile.avatar,
             thumbUri: profile.avatar,
+            thumbRect: null,
```

Comment:
this is now per-image because we need to support "open one image, swipe to next one, close" and animate out the other one. i.e. we need to know _each_ image's dimensions.